packages/language-server/src/plugins/css/CSSClassDefinitionLocator.ts

```diff
@@ -0,0 +1,167 @@
+import { Position, Range } from 'vscode-languageserver';
+import { SvelteDocumentSnapshot } from '../typescript/DocumentSnapshot';
+import { Document } from '../../lib/documents';
+import { SvelteNode } from '../typescript/svelte-ast-utils';
+export class CSSClassDefinitionLocator {
+    initialNodeAt: SvelteNode;
+    constructor(
+        public tsDoc: SvelteDocumentSnapshot,
+        public position: Position,
+        public document: Document
+    ) {
+        this.initialNodeAt = this.tsDoc.svelteNodeAt(this.position) as SvelteNode;
+    }
+
+    public GetCSSClassDefinition() {
```

Comment:
The public methods don't need `public` in front, at least that's the code style in this repo. Also please use lowercase-first method names.

Reply:
habit from another language, thanks for noting